crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -7,29 +7,44 @@ use ruff_python_ast::node::AnyNodeRef;
 
 use crate::builders::parenthesize_if_expands;
 use crate::comments::{dangling_comments, CommentLinePosition};
-use crate::expression::parentheses::{
-    parenthesized, NeedsParentheses, OptionalParentheses, Parentheses,
-};
+use crate::expression::parentheses::{parenthesized, NeedsParentheses, OptionalParentheses};
 use crate::prelude::*;
 
 #[derive(Eq, PartialEq, Debug, Default)]
 pub enum TupleParentheses {
-    /// Black omits parentheses for tuples inside of comprehensions.
-    Comprehension,
-
     /// Effectively `None` in `Option<Parentheses>`
```

Comment:
This comment is outdated now. Can we document how this "layout" looks like (e.g. how is it different from `Preserve`?

Reply:
Yes. `Default` is the strategy that is used in the majority of tuple usages. We should either document how `Default` and `Preserve` differ **or** merge the two.

Reply:
I'm struggling to communicate how `Default` and `Preserve` could be different for tuples specifically, because semantically they represent the same thing to me. "Tuples by default will *preserve* their parentheses".

So I'm leaning towards merging.

Reply:
So I decided that my interpretation of default tuple parentheses strategy was incomplete. A complete definition is probably closer to "Tuples by default will preserve their parentheses, but in some cases parentheses may also be introduced".

Now if that's a *good default* is a different question (I think it's fine, just more complicated), but I added some of this verbiage in 1b3178e3abcb45f09929f1569ed54b07f2a349e9